### PR TITLE
[BUG] Resolve in comment section 

### PR DESCRIPTION
**Pull request by ishan0805**
### What kind of change does this PR introduce:
This PR Resolves the issue of comments not properly displaying . Only Top 3 comments were displayed .
Additional button (view more comments) is added  . This button is displayed only when more the 3 comments are there to display.   
### Issue Number:
#399 
### Did you add tests for your code?
I have tested the code on emulator but didn't add any unit tests. 
### Snapshots/Videos:

https://user-images.githubusercontent.com/57336738/111895842-534a1680-8a3b-11eb-9f91-81300e7a964b.mp4

### CODE REVIEW

File: lib/views/pages/newsfeed/newsArticle.dart [hunk 1]
```diff
@@ -61,88 +60,112 @@ class _NewsArticleState extends State<NewsArticle> {
       String mutation =
           Queries().createComments(widget.post['_id'], commentController.text);
       Map result = await apiFunctions.gqlmutation(mutation);
+      //print("knfkafasnfak");
```
Comment (Sagar2366) [inline lib/views/pages/newsfeed/newsArticle.dart:63]: why this print statement required ?

**Comment by Sagar2366** [comment]
@ishan0805 really appreciate this work.
Can you please add unit test for this code just to make sure nothing else is breaking because of this changes ?

File: lib/views/pages/newsfeed/newsArticle.dart (hunk 1 above)
Comment (ishan0805) [reply lib/views/pages/newsfeed/newsArticle.dart:63]: actually there was some error in posting the comments . Therefore I was testing that the results are send properly or not .
After that I commented it.

**Comment by ishan0805** [comment]
Fine I will add them

**Comment by Sagar2366** [comment]
> Fine I will add them

Also resolve conflicts on this PR.

**Comment by ishan0805** [comment]
Tests are in file newsArticle_test.dart

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**